modules/xfem/test/tests/solid_mechanics_basic/prop_czm.i

```diff
@@ -0,0 +1,220 @@
+[GlobalParams]
```

Comment:
I just realized that none of the tests in this directory were added to the 'tests' file, so they're not being run.

Reply:
Right, I didn't add them because I didn't want to gold a test if I cannot judge whether the results make sense.